### DISCUSSION TITLE
Source of the transports data

### DISCUSSION THREAD

**Discussion by augustin64** [grote/Transportr#792] [Q&A] [answered]
From where is the transport data extracted ? I want to know how I can contribute to the project on this side, but couldn't find it myself in the repo...

**Comment by ialokim** [comment, marked as answer]
As explained on https://transportr.app/contribute/, Transportr relies on [PTE](/schildbach/public-transport-enabler) as its gateway to the data sources. If you'd like to contribute, please refer to this site, where the different processes of adding new regions is sketched, too.
